pkgs/applications/window-managers/i3/i3ipc-autotiling.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, python3Packages }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "i3ipc-autotiling";
+  version = "d521cd6d26d0b3aadbd283fe4165235b5df386f2";
```

Comment:
the nixpkgs manual says you should use the date as version if upstream does not tag versions.
See https://nixos.org/nixpkgs/manual/#sec-package-naming